source/api/commands/state.md

```diff
@@ -0,0 +1,148 @@
+---
+title: state
+
+---
+
+Get and set state _synchronously_.
+
+# Syntax
+
+```javascript
+.state()
+.state(key)
+.state(key, value)
+```
+
+## Usage
+
+### Getting
+
+```javascript
+var window = cy.state("window")
+```
+
+```javascript
+var location = cy.state("location")
+```
+
+You can also retrieve all of the existing state synchronously:
+
+```javascript
+var allState = cy.state()
+```
+
+### Setting
+
+```javascript
+cy.state("ids", [1, 2, 3])
+```
+
+The state with the key `ids` can subsequently be accessed synchronously.
+
+```javascript
+var ids = cy.state("ids") // [1, 2, 3]
+```
+
+## Arguments
+
+**{% fa fa-angle-right %} key**  ***(String)***
+
+The `key` of the `value` that is being get or set.
+
+**{% fa fa-angle-right %} value**  ***(any)***
```

Comment:
`Steer Needed`

What's the "type" used to describe anything? I used the TypeScript convention of `any` there because I couldn't find an example in the existing docs to crib off. Maybe `Object`?